docs/ref/forms/validation.txt

```diff
@@ -205,6 +205,15 @@ with ``code``\s and ``params`` but a list of strings will also work::
         _('Error 2'),
     ])
 
+If you detect multiple errors in multiple fields in your form, for instance
```

Comment:
This will **only** work in `Form.clean()`, so "for instance" is a little light here.